src/platform/site-wide/va-footer/components/LanguageSupport.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+
+function LanguagesListTemplate() {
+  return (
+    <ul>
+      {[
+        {
+          onThisPage: 'En esta página',
+          label: 'Español',
+          suffix: '-esp/',
+          lang: 'es',
+        },
+        {
+          suffix: '-tag/',
+          label: 'Tagalog',
+          onThisPage: 'Sa pahinang ito',
+          lang: 'tl',
+        },
+        {
+          label: 'Other languages',
+          suffix: '/',
+          lang: 'en',
+        },
+      ].map((link, i) => (
+        <li key={i}>
+          <a
+            href={link.href}
```

Comment:
they aren't ready yet, should have marked this PR as a draft. waiting for the links from product.

Reply:
since we have a feature flag for this, was advised by @SKasimow that we can merge this in without the hrefs and enable feature flag once hrefs are ready